txtdirect.go

```diff
@@ -279,6 +279,15 @@ func query(zone string, ctx context.Context, c Config) ([]string, error) {
 	return txts, nil
 }
 
+func isIP(host string) bool {
+	hostSlice := strings.Split(host, ".")
+	_, err := strconv.Atoi(hostSlice[len(hostSlice)-1])
```

Comment:
That's an interesting way of checking for an IP.
Not saying it's incorrect, but this doesn't cover IPv6.

So let's add IPv6 within the tests and check, if this still holds true.
Maybe there is also an underlying issue with the fallback not being triggered for IPs even so it should have.

Reply:
What I mean. I think the fallback should have triggered due to the failing lookups already so it might be a bug in the code and not that specific to IPv4 or IPv6 addresses.